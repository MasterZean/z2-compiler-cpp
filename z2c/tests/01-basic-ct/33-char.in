'a'
'7'
'Ş'
'ş'
'猫'
'ᾀ'
'ǲ'
'ǳ'
'Ǳ'