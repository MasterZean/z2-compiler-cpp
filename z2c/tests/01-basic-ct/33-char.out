'a';
'7';
350;
351;
29483;
8064;
498;
499;
497;
